pkgs/development/octave-modules/vrml/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildOctavePackage
+, stdenv
+, fetchurl
+# Octave dependencies
+, linear-algebra
+, miscellaneous
+, struct
+, statistics
+# Runtime dependencies
+, freewrl
+}:
+
+buildOctavePackage rec {
+  pname = "vrml";
+  version = "1.0.13";
+
+  src = fetchurl {
+    url = "https://octave.sourceforge.io/download.php?package=${pname}-${version}.tar.gz";
+    sha256 = "1mx93k150agd27mbzvds13v9z0x36j68hwpdvlvjmcl2fga5fly4";
+  };
+
+  propagatedBuildInputs = [
+    freewrl
+  ];
+
+  requiredOctavePackages = [
+    linear-algebra
+    miscellaneous
+    struct
+    statistics
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://octave.sourceforge.io/${pname}/index.html";
```

Comment:
I would prefer not using pname here to easily click on the link but not that important.

Reply:
Other than hard-coding in the `${pname}` of the package, I don't know an alternative to make this URL easier to resolve. Unless, that's what you're suggesting.

Reply:
I suggest exactly that. It makes it easier to open links from the source. For search.nixos.org it does not matter.